src/docs/concepts-api-sdk/api-reference/index.md

```diff
@@ -2,7 +2,7 @@
 title: API Reference
 slug: api
 updated: 2021-10-05
-category: references
+category: concepts-api-sdk
```

Comment:
Should we have something in References for APIs? Based on the notion that references are the single source of truth and the most technically oriented visitors search directly in there, I'd suggest moving this article and the SDKs article into References

Reply:
I see what you mean – these articles could easily fall under the references category. However, I do see a benefit in keeping all API and SDK related content grouped together:
<img width="233" alt="Screenshot 2022-04-08 at 11 31 40" src="https://user-images.githubusercontent.com/89124964/162397477-dc4fceca-88b6-47f7-a45c-7f83833df76b.png">

These articles are also practically linking to our API instead of being a pure technical reference. But I also see the benefits of moving them to references. What are your thoughts?

Reply:
I think we could go either way here, and if we decide to keep these under Concepts, I'd suggest having something about the APIs visible in references as well, if only for findability